apps/geolibre-desktop/src/lib/pyodide/console_api.py

```
@@ -0,0 +1,226 @@
+        return self._info.get("name")
+
+    @property
+    def type(self):
+        return self._info.get("type")
+
+    @property
+    def visible(self):
+        return self._info.get("visible", True)
+
+    @visible.setter
+    def visible(self, value):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `visible`

Add return type annotation: `None`

(ANN202)